rpi_backlight/cli.py

```diff
@@ -120,6 +124,18 @@ def main():
                     backlight.power = True
                 with backlight.fade(duration=args.duration):
                     backlight.brightness = 100
+        elif args.set_power == "on":
+            if backlight.power:
+                return
```

Comment:
Why does this have a check and "off" doesn't?

Reply:
I set this so it does not try to power on if already on (This will prevent the brightness from setting to 100 if brightness is already turned up (i.e. 40) and you try to power on)
This fixes https://github.com/linusg/rpi-backlight/issues/44

Reply:
on/off should be consistent with toggle